cmd/snap-repair/staging.go

```diff
@@ -76,6 +77,12 @@ func init() {
 		panic(fmt.Sprintf("cannot decode trusted account-key: %v", err))
 	}
 	if snapdenv.UseStagingStore() {
-		trustedRepairRootKeys = append(trustedRepairRootKeys, repairRootAccountKey.(*asserts.AccountKey))
+		trustedRepairRootKeys = append(trustedRepairRootKeys, repairRootAccountKey.(*asserts.AccountKey), systestkeys.TestStoreAccountKey.(*asserts.AccountKey))
```

Comment:
this is not correct,   the build constraint of this file are ORed,    but test keys should be added only if withtestkeys is defined  so as asserts/sysdb  we need a dedicated testkeys.go or testing.go file

Reply:
Good point, I moved adding the testrootorg keys to a separate file that is only used with the `withtestkeys` tag.